src/index.ts

```diff
@@ -1,12 +1,16 @@
 import { useState, useEffect } from 'react';
 import { RefTaggerSettings } from './interfaces';
 
-const addScript = (setScriptAdded: React.Dispatch<React.SetStateAction<boolean>>): void => {
+const addScript = (
+  setScriptAdded: React.Dispatch<React.SetStateAction<boolean>>,
+  language: string
+): void => {
   setScriptAdded(true);
+  const languageSuffix = language && language !== 'en' ? `.${language}` : '';
```

Comment:
Sorry to randomly come across your PR, but I had a quick suggestion. Perhaps remove `language !== 'en'` and leave the default language up to `RefTagger.js`. It also makes the logic more maintainable in the future.

Great PR, by the way :blush::+1:

```suggestion
  const languageSuffix = language ? `.${language}` : '';
```

Reply:
@jameschensmith I agree that that would be much cleaner, however, the issue is that `https://api.reftagger.com/v2/RefTagger.en.js` is not (currently) supported. English requires no language suffix and the other languages require their respective language suffix.

Reply:
@multiwebinc, that is why I'm suggesting setting the type to the supported languages (`'es' | 'fr' | 'de'`), then removing the language-specific check in `src/index.ts` (the suggested change mentioned above).

That way, once `en` is supported, all that is needed is to update the type to be `'en' | 'es' | 'fr' | 'de'`. Anyone who wants to use `en` currently continues to not pass the language as a property until it's supported. It's quite a clean solution.

In any case, this is only a suggestion, so you can choose whichever you think is best :slightly_smiling_face:

Reply:
Oh, ok. I got you now. I slightly misread. Either way is fine for me. However, would it cause confusion to people that omitting the language attribute is how you set it to English? I think the most important thing is that it is as clear as possible to the person implementing the extension rather than having the code being slightly cleaner. What do you think?

Reply:
With `language` property being newly added, I would think that no one is passing `language` to their React component currently. So, at the moment this PR is backwards-compatible. Big thumbs up :+1: Now, because it's TypeScript, someone would have to go out of their way to ignore compile errors / IDE flagging if they passed any other language than the supported locales (currently, that's `'es' | 'fr' | 'de'`). Something you can also do is add a doc comment to the field for that extra clarity (documenting the interfaces would be :star2:).

Think about it this way. Once `en` is added as a language suffix, that `&& language !== 'en'` was going to be removed anyways, right? My biggest concern with the current state was the assumption that the default locale file (`RefTagger.js`) will always be in English. It may always be, but it's best to aim to be resilient in the worst-case scenarios.

Reply:
Hey @multiwebinc, just wanted to check in on this PR. You don't have to take my suggestions. If you do, though, I wouldn't mind making the changes (either as a PR to your forked branch, or a separate PR, etc.).

Also, it appears `RefTagger.en.js` is now supported! So, the code change we spoke about in this chain can be made now without any concerns :slightly_smiling_face: Hope you're doing well!